app/policies/user_policy.rb

```diff
@@ -14,6 +14,10 @@ def create?
     user.admin?
   end
 
+  def update?
+    user.admin?
```

Comment:
That's the way, but we need to be more flexible here, for example, if a user wants to update their phone for example, with permission this way, he/she will need to ask for an admin to do it. Also, we need to call this action inside the [controller](https://github.com/comarev/comarev/blob/master/app/controllers/users_controller.rb#L20) :)

Reply:
That change was removed! Finally found the real source of error!

Reply:
I believe that the  `authorize` method was already implemented, wasn't it?

https://github.com/comarev/comarev/blob/12e61426208d78071f5d250bd848bb4b54a731d4/app/controllers/users_controller.rb#L35

Reply:
> I believe that the `authorize` method was already implemented, wasn't it?
> 
> https://github.com/comarev/comarev/blob/12e61426208d78071f5d250bd848bb4b54a731d4/app/controllers/users_controller.rb#L35

Nice catch, it is